Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -53,6 +53,7 @@ async function getKubeConfigFile(): Promise<string> {
         var configFilePath = getKubeConfigFilePath();
         tl.debug(tl.loc("KubeConfigFilePath", configFilePath));
         fs.writeFileSync(configFilePath, config);
+        fs.chmodSync(configFilePath, 0o700);
```

Comment:
Could use string value for mode like in [other places](https://github.com/microsoft/azure-pipelines-tasks-common/blob/main/kubernetes-common-v2/kubectlutility.ts#L49)